pymatgen/electronic_structure/cohp.py

```diff
@@ -332,25 +361,32 @@ def as_dict(self):
 
         return dct
 
-    def get_cohp_by_label(self, label, summed_spin_channels=False):
-        """Get specific COHP object.
+    def get_cohp_by_label(
+        self,
+        label: CohpLabel,
+        summed_spin_channels: bool = False,
+    ) -> Cohp:
+        """Get specific Cohp object by label.
 
         Args:
-            label: string (for newer Lobster versions: a number)
```

Comment:
@naik-aakash Can I have a quick comment on the expected type of `label` as well? 

As per the docstring as could be either `str` or `number`(I assume `int`), but the implementation doesn't seem to take `int` into account (`label.lower()`)

Reply:
Hi @DanielYang59 , The files have both kinds of labels a number but in string form '1' , '2' etc, and there is also an label `average` in the cohpcar files. Hope this clarifies

Reply:
Great! Thanks!